packages/node_modules/@webex/plugin-meetings/src/common/collection.js

```diff
@@ -0,0 +1,95 @@
+import {isString} from 'lodash';
+
+import {MEETINGS} from '../constants';
+
+/**
+ * @class Collection
+ */
+export default class Collection {
+  namespace = MEETINGS;
+
+  /**
+   * @param {String} property
+   * @memberof Collection
+   */
+  constructor(property) {
+    if (!property || !isString(property)) {
+      throw new Error('Collection should be initialized with a property string that will be used to store data at.');
+    }
+    /**
+     * The property name for what collection will be stored, i.e., this.meetingInfos, this.meetings, etc.
+     * @instance
+     * @type {String}
+     * @public
+     * @memberof Meetings
+     */
+    this.propertyName = property;
+    /**
+     * The actual object for the collection
+     * @instance
+     * @type {Object}
+     * @public
+     * @memberof Meetings
+     */
+    this[this.propertyName] = {};
+  }
+
+  /**
+   * @param {String} id ID of the thing stuffed into the collection at id location
+   * @returns {Any} returns whatever is being stuffed into the collection
+   * @public
+   * @memberof Collection
+   */
+  get(id) {
+    if (this[this.propertyName] && this[this.propertyName][id]) {
```

Comment:
it would technically return `undefined` instead of `null`(still both falsey), but you could change this method to:
```
get(id) {
  return this[this.propertyName] && this[this.propertyName][id]
}
```

Reply:
would rather return null, but could do a ternary